pkg/reconciler/instances/connectivityproxy/command_test.go

```diff
@@ -145,8 +145,8 @@ func TestCommandInstall(t *testing.T) {
 
 		err := commands.Install(actionContext, secret)
 		require.Equal(t, map[string]interface{}{
-			"key-1": []byte("value-1"),
-			"key-2": []byte("value-2"),
+			"binding.key-1": []byte("value-1"),
+			"binding.key-2": []byte("value-2"),
```

Comment:
It'd be superb to also add the JSON marshalled values to see if the implementation works correctly in these cases

Reply:
Outdated.